packages/multi-entry/test/test.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-bitwise */
+
+import test from 'ava';
+import { rollup } from 'rollup';
+
+import { getCode } from '../../../util/test';
+
+import multiEntry from '../';
+
+test('takes a single file as input', async (t) => {
+  const bundle = await rollup({ input: 'test/fixtures/0.js', plugins: [multiEntry()] });
+  const code = await getCode(bundle);
+  t.truthy(~code.indexOf('exports.zero = zero;'));
```

Comment:
These bitwise operators are hard to read, I think `includes` would be better.
```suggestion
  t.true(code.includes('exports.zero = zero;'));
```

Reply:
you kids and your fancy new es6 methods

![old man](https://i.kym-cdn.com/entries/icons/facebook/000/019/304/old.jpg)